libraries/lib-wave-track/WaveClip.h

```diff
@@ -474,6 +501,9 @@ class WAVE_TRACK_API WaveClip final :
    double mSequenceOffset { 0 };
    double mTrimLeft{ 0 };
    double mTrimRight{ 0 };
+   double mClipStretchRatio = 1.;
+   std::optional<double> mRawAudioTempo;
+   std::optional<double> mProjectTempo;
```

Comment:
Please comment these new data members and also the old ones.

The invariant assumed for `mSequenceOffset`, `mTrimLeft`, `mTrimRight` are that they are always realtime durations.

But the sample interval `1 / mRate` is NOT necessarily realtime duration.

`mClipStretchRatio` is used in `GetStretchRatio` which computes the factor, by which the sample interval is multiplied, to get a realtime duration.